exp-player/addon/components/exp-overview/component.js

```diff
@@ -29,6 +29,12 @@ var generateValidators = function(questions) {
   return validators;
 };
 
+var getQuestion8Options = function() {
+    var scale = range(1, 10);
+    scale.push('survey.sections.1.questions.8.options.preferNoAnswer');
```

Comment:
How will this field be represented in the exported data?

For example, if they analyze it always expecting a number, what will their data analysis program do if it sees a string? Is there a more standardized way of representing missing data for their programs? If so, should we consider storing the response that way instead of storing a string?